replay/models/__init__.py

```diff
@@ -3,21 +3,31 @@
 
 - distributed models built in PySpark
 - neural networks build in PyTorch with distributed inference in PySpark
-- wrappers for commonly used recommender systems libraries and\
-models with non-distributed training and distributed inference in PySpark.
+- wrappers for commonly used recommender systems libraries and
+    models with non-distributed training and distributed inference in PySpark.
 """
 
+from replay.models.admm_slim import ADMMSLIM
 from replay.models.als import ALSWrap
 from replay.models.association_rules import AssociationRulesItemRec
 from replay.models.base_rec import Recommender
-from replay.models.cat_pop_rec import CatPopRec
-from replay.models.cluster import ClusterRec
+from replay.models.base_torch_rec import TorchRecommender
```

Comment:
Need to update imports list, the most of the changes are in the experimental submodule now